package.json

```diff
@@ -1,16 +1,14 @@
 {
-  "name": "electroblocks-tray-app",
+  "name": "electron_app",
   "version": "1.0.0",
+  "type": "module",
   "main": "main.js",
   "scripts": {
-    "start": "electron .",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "electron ."
   },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "description": "",
-  "devDependencies": {
-    "electron": "^34.2.0"
+  "dependencies": {
+    "electron": "^35.0.1",
+    "express": "^4.18.2",
```

Comment:
Decide between ws adn sockets.io for the library.  WS library is simpler to use but less secure